client/src/components/Pets/PetStatus/PostPetStatus.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import useStyles from './useStyles';
+import { Box, TextField, Button } from '@material-ui/core';
+import { Formik } from 'formik';
+const PostPetStatus = (): JSX.Element => {
+  const classes = useStyles();
+  const handleSubmit = ({ description }: { description: string }) => {
+    console.log('submit');
```

Comment:
No need for this console log here

Reply:
Hi, as the function can not be empty, can I just keep this console log here?  I will remove it at the next integration ticket